WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -65,6 +65,7 @@
 static NSDictionary *EnabledButtonBarStyle;
 
 static void *ProgressObserverContext = &ProgressObserverContext;
+static void *DateChangeObserverContext = &DateChangeObserverContext;
```

Comment:
I'd recommend using `static void const` so that these context pointers can't ever be altered. It's a nitpick though as I've seen both ways.

```objc
static void * const ProgressObserverContext = (void*)&ProgressObserverContext;
static void * const DateChangeObserverContext = (void*)&DateChangeObserverContext;
```